spec/integration/pr/test-pr-1216.spec.js

```diff
@@ -0,0 +1,10 @@
+const ExcelJS = verquire('exceljs');
+
+describe('github issues', () => {
+  it('pull request 1216 - The worksheet should not be undefined', async () => {
```

Comment:
shouldn't this be PR #1220 ? PR looks good otherwise

Reply:
@alubbe I update the test case. thanks :).